src/tests/accessibility/accessibility.spec.ts

```diff
@@ -0,0 +1,38 @@
+import AxeBuilder from 'axe-webdriverjs';
+import chromeDriver from 'chromedriver';
+import webdriver, { ThenableWebDriver } from 'selenium-webdriver';
+import chrome from 'selenium-webdriver/chrome';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 300000;
+const TEST_TIMEOUT_MS = 300000;
+
+describe('Graph Explorer', () => {
+  let driver: ThenableWebDriver;
+
+  beforeAll(async () => {
+    chrome.setDefaultService(new chrome.ServiceBuilder(chromeDriver.path).build());
+
+    driver = new webdriver
+    .Builder()
+    .withCapabilities(webdriver.Capabilities.chrome())
+    .build();
+  }, TEST_TIMEOUT_MS);
+
+  afterAll(async() => {
+    return driver && driver.quit();
+  }, TEST_TIMEOUT_MS);
+
+  beforeEach(async() => {
+    await driver.manage().setTimeouts( { implicit: 0, pageLoad:
+      60000, script: TEST_TIMEOUT_MS });
+    await driver.get('https://developer.microsoft.com/en-us/graph/graph-explorer/preview');
```

Comment:
The url here leads to the preview site. Can we try get it to read the actual live site that is without the /preview?

Reply:
okay yeah sure

Reply:
Yeah I also think that would be a better approach
let me look into how we can better run this test